samples/bluetooth/broadcast_audio_sink/src/main.c

```diff
@@ -149,35 +148,61 @@ static void lc3_decoder_thread(void *arg1, void *arg2, void *arg3)
 {
 	while (true) {
 		k_sem_take(&lc3_decoder_sem, K_FOREVER);
-#if defined(CONFIG_USB_DEVICE_AUDIO)
-		int err = 0;
-		enum bt_audio_location channels;
-		struct broadcast_sink_stream *stream_for_usb = &streams[0];
 
+#if defined(CONFIG_USB_DEVICE_AUDIO)
 		/* For now we only handle one BIS, so always only decode the first element in
-		 * streams.
+		 * streams
 		 */
-		do_lc3_decode(&streams[0]);
-
-		err = bt_audio_codec_cfg_get_chan_allocation(stream_for_usb->stream.codec_cfg,
-							     &channels);
-		if (err != 0) {
-			printk("Could not get channel allocation (err=%d)\n", err);
-			continue;
-		}
+		struct broadcast_sink_stream *stream_usb = &streams[0];
+#endif /* #if defined(CONFIG_USB_DEVICE_AUDIO) */
```

Comment:
Suggest to move the declaration to line 161 to avoid 2 x `#if defined(CONFIG_USB_DEVICE_AUDIO)`